tests/testthat/helper-functions.R

```diff
@@ -195,24 +195,13 @@ getStyleValue <- function(html, xpath, style.name) {
 }
 
 ## testthat there is no warning generated by a piece of code.
-gives_no_warning <- function(){
-  function(expr) {
-    warnings <- evaluate_promise(expr)$warnings
-    s <- ifelse(length(warnings)==1, "", "s")
-    expectation(
-      length(warnings) == 0,
-      paste0("created ", length(warnings),
-             " warning", s, ": ",
-             paste(warnings, collapse=", ")),
-      "no warnings given"
-    )
-  }
-}
-expect_no_warning <- function(object, ..., info=NULL, label=NULL){
-  if (is.null(label)) {
-    label <- testthat:::find_expr("object")
-  }
```

Comment:
that is probably one reason why there is a conflict with master --- we have already merged changes to `expect_no_warning` https://github.com/tdhock/animint/blob/master/tests/testthat/helper-functions.R#L228

I will pull changes from master into this branch and try to resolve the conflicts.